pax-manifest/src/utils.rs

```diff
@@ -10,6 +10,12 @@ use pest_derive::Parser;
 pub struct PaxParser;
 
 pub fn parse_value(raw_value: &str) -> Result<ValueDefinition> {
+    if raw_value.is_empty() {
```

Comment:
Let's throw an error here. We cannot set an empty string in the manifest, that would fail at deserialization time